doc/design.md

```diff
@@ -43,7 +43,7 @@ The recipient can calculate the sender's contact ID based on the public key, and
 
 The recipient user has the choice to accept or reject that request. A rejected public key may be added to a blacklist and rejected automatically from future requests.
 
-For more detail, see section 8 of the [protocol documentation](https://github.com/ricochet-im/ricochet/blob/master/doc/protocol.txt).
+For more detail, see section 8 of the [protocol documentation](https://github.com/ricochet-im/ricochet/blob/master/doc/protocol.md).
```

Comment:
This should probably be changed to https://github.com/ricochet-im/ricochet/blob/master/doc/protocol.md#contact-request-channel